docs/data-sources/user.md

```diff
@@ -0,0 +1,58 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "grafana_user Data Source - terraform-provider-grafana"
+subcategory: ""
+description: |-
+  Official documentation https://grafana.com/docs/grafana/latest/manage-users/server-admin/server-admin-manage-users/HTTP API https://grafana.com/docs/grafana/latest/http_api/user/
+  This resource uses Grafana's admin APIs for creating and updating users which
+  does not currently work with API Tokens. You must use basic auth.
+---
+
+# grafana_user (Data Source)
+
+* [Official documentation](https://grafana.com/docs/grafana/latest/manage-users/server-admin/server-admin-manage-users/)
+* [HTTP API](https://grafana.com/docs/grafana/latest/http_api/user/)
+
+This resource uses Grafana's admin APIs for creating and updating users which
+does not currently work with API Tokens. You must use basic auth.
```

Comment:
Can we check somehow in the code that you're using Basic Auth and return a specific error or will it just fail?

Reply:
Hmm I copied this from the `grafana_user` resource. Let me check if it's still relevant

Reply:
Hmm, it's still relevant but I think I'd rather rely on the API 403 (in case it ever changes)